crescent/internal/app_command.py

```diff
@@ -114,6 +127,20 @@ def set_id(self, _id: UndefinedOr[Snowflakeish]) -> AppCommand:
         self.id = _id
         return self
 
+    async def create(
+        self,
+        rest: RESTClient,
+        application: SnowflakeishOr[PartialApplication],
+        /,
+        *,
+        guild: UndefinedOr[SnowflakeishOr[PartialGuild]] = UNDEFINED,
+    ) -> PartialCommand:
+        """
+        This method isn't used but someone made it an abstract method in CommandBuilder...
+        """
+        _LOG.warning("Method `create` is not implemented for `AppCommand`.")
+        return  # type: ignore
```

Comment:
I don't like this... but I can't think of a better approach really.

Reply:
I changed it to an exception so i didn't need to deal with return type